CefSharp.Test/CefSharp.Test.csproj

```diff
@@ -109,6 +109,10 @@
     <Compile Include="Wpf\WpfBrowserBasicFacts.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\CefSharp.BrowserSubprocess\CefSharp.BrowserSubprocess.csproj">
```

Comment:
It's copied using a `target`, https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Native.props#L3 like the other projects in the solution.

There should be no need for a reference.

Reply:
This is required so `msbuild` can calculate the build order correctly. Without this the `CefSharp.Test` project might (and is) built before the `CefSharp.BrowserSubprocess` project and therefore copies nothing. That's why the [`OffScreen` tests failed](https://ci.appveyor.com/project/cefsharp/cefsharp/builds/31114188/tests).

But I can change that to be a solution dependency instead of a project reference.

Note: this is also in the wpf sample project:
https://github.com/cefsharp/CefSharp/blob/749ec15972d332ce0fc59f3785184f884df748ae/CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj#L195-L198

Reply:
A project dependency would be preferable,  the WPF example should have the reference removed for consistency.